testData/preferences/condTest.json

```diff
@@ -0,0 +1,43 @@
+{
+    "flat": {
+        "contexts": {
+            "gpii-default": {
+                "name": "Default preferences",
+                "preferences": {
+                    "http://registry.gpii.net/common/fontSize": 12
+                },
+                "metadata" : [
+                    {
+                        "message" : {
+                            "de" : {
+                                "learnMore" : "http://cloud4all.info",
+                                "message" : "blablabla DE"
```

Comment:
German translation

Ihr bevorzugtes technisches Hilfsmittel ist auf diesem Gerät nicht vorhanden. Als Ersatz haben wir ein anderes Hilfsmittel gestartet.

Note: "vorhanden" conveys the meaning "present" or "installed". If you want to say that it does not exist on the current device "verfügbar" is better.